editions/test/tiddlers/tests/test-filters.js

```diff
@@ -1060,6 +1060,9 @@ Tests the filtering mechanism.
 			expect(wiki.filterTiddlers("[[Hello There]] [[GettingStarted]] +[format:titlelist[]]").join(" ")).toBe("[[Hello There]] GettingStarted");
 			expect(wiki.filterTiddlers("[title[Hello There]] +[format:titlelist[]]").join(" ")).toBe("[[Hello There]]");
 			expect(wiki.filterTiddlers("[title[HelloThere]] +[format:titlelist[]]").join(" ")).toBe("HelloThere");		
+			expect(wiki.filterTiddlers("0 +[format:timestamp[]]").join(",")).toBe("19700101000000000");
+			expect(wiki.filterTiddlers("1603188514443 +[format:timestamp[]]").join(",")).toBe("20201020100834443");
```

Comment:
Something like this would be nice too `"1603188514443" +[format:timestamp[YYYY-0MM-0DD 0hh:0mm:0ss.0XXX]]`

Reply:
I'm not sure that any formats without "[UTC]" in their definition should be used for these checks, as the results will be time-zone dependent...
Putting "[UTC]" in the date string requires an external variable, which I rather not bothered with.